src/todoItem/index.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { FC } from "react";
+import { Item, RemoveButton } from "../App.styled";
+type ComponentProps = {
+  id?: number;
```

Comment:
`id` doesn't need to be optional

```suggestion
  id: number;
```

Same with `name` and `done`